### PR TITLE
feat: add hint to search for hd-contacts by postal code or city

### LINKED ISSUE #632: Hint to search by postal code and city
Add a note to the message recipient field that it is also possible to search by ZIP code and city.

### PR DESCRIPTION
**Pull request by mad-nuts**
Resolves #632

### CODE REVIEW

File: iris-client-fe/src/views/iris-message-create/iris-message-create.view.vue [hunk 1]
```diff
@@ -22,11 +22,16 @@
...
             :class="{
               'is-loading': recipientsLoading,
               'is-empty': recipients.length <= 0,
             }"
-          ></v-autocomplete>
+          />
+          <p class="text-caption mt-2">
+            Tippen Sie die <strong>Postleitzahl</strong> oder den
+            <strong>Ort</strong> ein, um nach dem Empfänger zu suchen.
+          </p>
```
Comment (jekutzsche) [inline iris-client-fe/src/views/iris-message-create/iris-message-create.view.vue:34]: Is there no mechanism for I18N in the FE?

Doesn't this text suggest too little that it is in addition to the GA name?

Comment (mad-nuts) [reply iris-client-fe/src/views/iris-message-create/iris-message-create.view.vue:34]: There is no 18n support in the frontend at the moment. The health department name will show up as a selectable option in the dropdown menu if any match is found. No need for the user to type in the GA name in addition to the zip-code / city.

Comment (mad-nuts) [reply iris-client-fe/src/views/iris-message-create/iris-message-create.view.vue]: I have added "Bezeichner des Gesundheitsamtes" to the hint text as we do not have a display name for the health department selection at the moment.

**Review by jekutzsche — APPROVED** [review]

**Pull request merged**